cmd/otelcol/config/collector/full_config_linux.yaml

```diff
@@ -167,6 +167,14 @@ receivers:
   fluentforward:
     endpoint: 0.0.0.0:8006
 
+  # Enables scripted inputs receiver
+  # Full configuration here: https://github.com/signalfx/splunk-otel-collector/tree/main/internal/receiver/scriptedinputsreceiver
+  scripted_inputs/cpu:
```

Comment:
just occurred to me this wouldn't be functioning until the component is added, so should be part of that addition PR imo.